### ISSUE TITLE
The Scrum Chatter App Crashes ~ bug

### ISSUE THREAD

**Issue by beckysophie** [caarmen/scrumchatter#65]
***Expected Behavior***

It should be able to allow users rotate their screen so as to properly use their screen to choose their preferred settings 

***Actual Behavior***

It crashes each time the screen is rotated while on the settings/theme menu.... 

***How to produce the bug***

Open the app, click on more options at the top right corner, select settings, click on theme Now rotate your screen, it would crash... 

***PS***

You should enable auto rotate, so as to view produce the bug 

* Operating system : Android 6.0
* Phone : Infinix hot s
* Scrum Chatter v1.6.4

***Recording of the bug is showned below......***

Watch the video below to see how it behaves.... 

https://youtu.be/yzP0dU4Z2YI

<br /><hr/><em>Posted on <a href="https://utopian.io/utopian-io/@beckysophie/the-scrum-chatter-app-crashes-bug">Utopian.io -  Rewarding Open Source Contributors</a></em><hr/>

**Comment by caarmen** [comment]
Thanks, I was able to reproduce this. Here's the stacktrace to help (myself) to debug this:
```
04-01 13:14:04.629  3033  3033 E AndroidRuntime: java.lang.IllegalStateException: Failure saving state: ListPreferenceDialogFragmentCompat{7461a9f #1 android.support.v7.preference.PreferenceFragment.DIALOG} has target not in fragment manager: GeneralPreferenceFragment{aeff7ec}
04-01 13:14:04.629  3033  3033 E AndroidRuntime:        at android.support.v4.app.FragmentManagerImpl.saveAllState(FragmentManager.java:2937)
04-01 13:14:04.629  3033  3033 E AndroidRuntime:        at android.support.v4.app.FragmentController.saveAllState(FragmentController.java:125)
04-01 13:14:04.629  3033  3033 E AndroidRuntime:        at android.support.v4.app.FragmentActivity.onSaveInstanceState(FragmentActivity.java:540)
04-01 13:14:04.629  3033  3033 E AndroidRuntime:        at android.support.v7.app.AppCompatActivity.onSaveInstanceState(AppCompatActivity.java:509)
04-01 13:14:04.629  3033  3033 E AndroidRuntime:        at android.app.Activity.performSaveInstanceState(Activity.java:1272)
04-01 13:14:04.629  3033  3033 E AndroidRuntime:        at android.app.Instrumentation.callActivityOnSaveInstanceState(Instrumentation.java:1289)
04-01 13:14:04.629  3033  3033 E AndroidRuntime:        at android.app.ActivityThread.callCallActivityOnSaveInstanceState(ActivityThread.java:4081)
04-01 13:14:04.629  3033  3033 E AndroidRuntime:        at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:4043)
04-01 13:14:04.629  3033  3033 E AndroidRuntime:        at android.app.ActivityThread.-wrap15(ActivityThread.java)
04-01 13:14:04.629  3033  3033 E AndroidRuntime:        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1351)
04-01 13:14:04.629  3033  3033 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:102)
04-01 13:14:04.629  3033  3033 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:148)
04-01 13:14:04.629  3033  3033 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:5415)
04-01 13:14:04.629  3033  3033 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
04-01 13:14:04.629  3033  3033 E AndroidRuntime:        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:725)
04-01 13:14:04.629  3033  3033 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:615)
```

**Comment by caarmen** [comment]
Info: the fix we should do is mentioned here :) https://github.com/Gericop/Android-Support-Preference-V7-Fix/issues/39

**Comment by caarmen** [comment]
@r4dixx if you want this one, it's yours :)

**Comment by r4dixx** [comment]
Cool! I'll have a look asap

**Comment by r4dixx** [comment]
Not so "asap" but it's done :smile:

**Comment by caarmen** [comment]
Fixed in version 1.6.6.
